core/modules/replica/DatabaseMySQL.cc

```diff
@@ -599,6 +622,20 @@ void Connection::_connectOnce() {
     enum mysql_protocol_type const prot_type = MYSQL_PROTOCOL_TCP;
     mysql_options(_mysql, MYSQL_OPT_PROTOCOL, &prot_type);
 
+    // This is required by 'LOAD DATA LOCAL INFILE ...' to allow ingesting files which
+    // are not directly seen by the MySQL server. The 'LOCAL' option would make a file
+    // local to a client opening this connection to be transferred to some temporary
+    // directory owned by the server. After that the file will get ingestd into
+    // the destination table.
+    //
+    // NOTES:
+    // - The server must be configured with global variable 'local_infile=1'
+    // - The temporary folder managed by the server is required to have enough space
+    //   to accommodate files received from the client.
```

Comment:
It seems like these notes would be worth putting in a header or in documentation (if they are duplicated here that's ok, but if they're only here they may be hard to find?)

Reply:
Good point! I will put this info into the class's header.